src/checkout/checkout-params.ts

```diff
@@ -2,6 +2,9 @@ export enum CheckoutIncludes {
     AvailableShippingOptions = 'consignments.availableShippingOptions',
     PhysicalItemsCategoryNames = 'cart.lineItems.physicalItems.categoryNames',
     DigitalItemsCategoryNames = 'cart.lineItems.digitalItems.categoryNames',
+    OrderPayments = 'payments',
```

Comment:
You'll need to define a new `OrderParams` interface. `CheckoutParams` is only for checkout endpoints.

Reply:
Also, you don't need to add `payments` to `OrderParams `.